scrnaseq.Rmd

```diff
@@ -37,26 +38,25 @@ plan("multiprocess") # by default, this uses all available cores; use "workers=4
 # Functional enrichment: enrichR
 
 # Source plotting functions
-path_to_git = "~/scrnaseq"
+path_to_git = "."
 source(paste0(path_to_git, "/R/functions_io.R"))
 source(paste0(path_to_git, "/R/functions_plotting.R"))
 source(paste0(path_to_git, "/R/functions_analysis.R"))
 source(paste0(path_to_git, "/R/functions_util.R"))
 
 # Knitr default options
-knitr::opts_chunk$set(echo=TRUE, cache=FALSE, message=FALSE, warning=FALSE, fig.width=10)
+knitr::opts_chunk$set(echo=TRUE, cache=FALSE, message=FALSE, warning=FALSE, tidy=FALSE, fig.width=10)
```

Comment:
`tidy=FALSE`? Dachte du möchtest es auf `TRUE` setzen?

Reply:
I think it was inspired by: https://www.stat.ubc.ca/~jenny/STAT545A/topic11_tidyCode.html